egs/librispeech/ASR/conv_transducer_stateless/convrnnt.py

```diff
@@ -0,0 +1,958 @@
+#!/usr/bin/env python3
+
+from typing import List, Optional, Tuple
+
+import torch
+from encoder_interface import EncoderInterface
+from scaling import (
+    ActivationBalancer,
+    BasicNorm,
+    DoubleSwish,
+    ScaledConv1d,
+    ScaledConv2d,
+    ScaledLinear,
+)
+from torch import Tensor, nn
```

Comment:
I suggest using `torch.Tensor` for consistency. Currently, you are using both `Tensor` and `torch.Tensor` for return types.

Reply:
OK, I will modify it.